features/webdriver.yml

```diff
@@ -1,16 +1,17 @@
-draft_date: 2024-08-12
 name: WebDriver
-description: TODO
+description: WebDriver is a remote control interface that enables introspection and control of user agents by out-of-process programs.
```

Comment:
Pinging folks who I know work on WebDriver to review this short description. 
@juliandescottes and @bwalderman, sorry if this out of context, you don't need to review the rest of the files here. I just want to make sure this short description is in line with how you think of webdriver, and more importantly that it's understandable by developers using it.

Reply:
Looks good to me, this is more or less a summary of the intro of the original WebDriver spec.

Reply:
thanks both!